client/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+
+version: "3"
+services:
+  pyrog_client:
+    image: arkhn/pyrog-client:latest
+    container_name: pyrog-client
+    build:
+      context: .
+    ports:
+      - $PORT:$PORT
```

Comment:
does this use the PORT variable from the .env.docker file  ?

Reply:
Yes :)

Reply:
:+1: